base_user_role/models/user.py

```diff
@@ -49,6 +54,14 @@ def write(self, vals):
         self.sudo().set_groups_from_roles()
         return res
 
+    def _get_enabled_roles(self):
```

Comment:
Could be renamed to `_get_applicable_roles` (something different from get_enabled_roles) to highlight that it's filtering on more than the field `is_enabled`.